src/game-support/weird-west-de.md

```diff
@@ -0,0 +1,18 @@
+# Weird West: Definitive Edition
+<!-- script:Aliases [
+    "Weird West: Definitive Edition"
```

Comment:
Write this alias without a colon as it should differ from the game title.

Reply:
All done, and thank you!